include/swift/AST/DiagnosticsSema.def

```diff
@@ -6649,6 +6649,9 @@ ERROR(availability_variadic_type_only_version_newer, none,
 NOTE(availability_guard_with_version_check, none,
      "add 'if #available' version check", ())
 
+NOTE(update_availability_attribute, none,
```

Comment:
Nit: Since the other diagnostic is named `availability_add_attribute`, maybe this one should be named `availability_update_attribute`.

Reply:
Ok, I will change the name in the next patch, thanks.